### PR TITLE
SvgXmlParser return parsed icon type, simplify extensions

### PR DESCRIPTION
**Pull request by egorikftp**
- closes: https://github.com/ComposeGears/Valkyrie/issues/256

Will be necessary for clipboard feature in IconPack mode

### CODE REVIEW

File: idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package io.github.composegears.valkyrie.util.extension
+
+import com.intellij.openapi.vfs.VirtualFile
+import io.github.composegears.valkyrie.parser.svgxml.util.isSvgExtension
+import io.github.composegears.valkyrie.parser.svgxml.util.isXmlExtension
+
+inline val VirtualFile.isSvg: Boolean
+    get() = extension?.isSvgExtension ?: false
```
Comment (Goooler) [inline idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt:8]:
```suggestion
    get() = extension?.isSvgExtension == true
```

File: idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+inline val VirtualFile.isXml: Boolean
+    get() = extension?.isXmlExtension ?: false
```
Comment (Goooler) [inline idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt:11]:
```suggestion
    get() = extension?.isXmlExtension == true
```

**Review by Goooler — APPROVED** [review]

File: idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt (hunk 1 above)
Comment (Goooler) [reply idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt:8]: Just a small thing. Looks good.

Comment (egorikftp) [reply idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/util/extension/VirtualFile.kt:8]: ok, will leave with false for now)
thanks for review 🙂

**Pull request merged**